htdocs/contact/class/contact.class.php

```diff
@@ -1868,7 +1868,7 @@ public function getContactRoles($element = '')
 	 * Updates all roles (default contact for companies) according to values inside the ->roles array.
 	 * This is called by update of contact.
 	 *
-	 * @return float|int
+	 * @return void|int
```

Comment:
Here the first return is empty maybe you could return 0 to specify that nothing was done. This done you can remove void return